### DISCUSSION TITLE
Sorting of fields to save one entry

### DISCUSSION THREAD

**Discussion by niels-przybilla** [kimai/kimai#4860] [General]
Hello Community,

from what I see when i select a project it auto select the customer.

So for the flow it would be nice to select first the project to save one click.
![Monosnap Kimai – Zeiterfassung 2024-05-15 16-03-33](https://github.com/kimai/kimai/assets/99684821/a0d37f70-a877-4c1a-8a4d-0ade331156e7)

**Comment by kevinpapst** [comment]
There is a data hierarchy customer => project => activity.
The customer is fully optional and only acts as a filter for the list of projects.
I don't get your proposal.

**Reply by niels-przybilla** [reply]
Ideas was just to prioritize the mandatory fields.
